mirrormanager2/lib/hostconfig.py

```diff
@@ -25,9 +25,14 @@
 
 import mirrormanager2.lib
 
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
if it doesn't matter, I think I'd rather have it be ``mirrormanager`` as not everyone may be running MM on a dedicated host